implementation.js

```diff
@@ -1,42 +1,114 @@
 'use strict';
 
-var isNaN = require('is-nan');
-
-var $isFinite = isFinite;
 var $Number = Number;
-var abs = Math.abs;
 
-var BINARY_32_EPSILON = 1.1920928955078125e-7; // 2 ** -23;
-var BINARY_32_MIN_VALUE = 1.1754943508222875e-38; // 2 ** -126;
-var BINARY_32_MAX_VALUE = 3.4028234663852886e+38; // 2 ** 128 - 2 ** 104
-var EPSILON = 2.220446049250313e-16; // Number.EPSILON
+var BINARY_32_MIN_VALUE = 1.401298464324817e-45; // 2 ** -(126 + 23)
+var BINARY_32_MIN_NORMAL = 1.1754943508222875e-38; // 2 ** -126
```

Comment:
Note that the previous implementation incorrectly called this constant `BINARY_32_MIN_VALUE`, although it is only the minimum *normal* value.